software/muscle/main.nf

```diff
@@ -0,0 +1,59 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process MUSCLE {
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::muscle=3.8.1551" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/muscle:3.8.1551--h7d875b9_6"
+    } else {
+        container "quay.io/biocontainers/muscle:3.8.1551--h7d875b9_6"
+    }
+
+    input:
+    path fasta
+
+    output:
+    path "*.afa"  , optional: true, emit: aligned_fasta
+    path "*.phyi" , optional: true, emit: phyi
+    path "*.phys" , optional: true, emit: phys
```

Comment:
As I understand from your documentation, these two belong together and only one of them is emitted at a time? 
In that case consider merging the output channel `"*.{phyi,phys}"`. 
Otherwise, please document every output channel separately.

Reply:
Apologies Gregor, my documentation was misleading.  MUSLCE can produce either or both phyi and phys output file in a single run - have updated meta.yml to reflect this